source/_ecosystem/ios/notifications/actions.markdown

```diff
@@ -0,0 +1,136 @@
+---
+layout: page
+title: "Actionable notifications"
+description: "Making push notifications a two way system"
+date: 2016-10-25 15:00:00 -0700
+sidebar: true
+comments: false
+sharing: true
+footer: true
+---
+
+Actionable notifications allow you to attach 1-4 custom buttons to a notification. When one of the actions is selected Home Assistant will be notified which action was chosen. This allows you to build complex automations.
+
+Examples of actionable notifications:
+
+- A notification is sent whenever motion is detected in your home while you are away or asleep. You can add an action to Sound Alarm. When tapped, Home Assistant is notified that the `sound_alarm` action was selected. You can add an automation to sound the burglar alarm whenever this event is seen.
+- Someone rings your front door bell. You can send an action to lock or unlock your front door. When tapped, a notification is sent back to Home Assistant upon which you can build automations.
+- Send a notification whenever your garage door opens with actions to open and close the garage.
+
+![Actions example](/images/ios/actions.png)
```

Comment:
Prefer if you use our img format:

```html
<p class='img'>
  <img src='/images/ios/actions.png' />
  Actionable notifications allow the user to send a command back to Home Assistant.
</p>
```

Also, please crop away all the not-needed part of the image, it takes a lot of height.